stdlib/Test/src/Test.jl

```diff
@@ -1279,7 +1279,7 @@ Body::UNION{FLOAT64, INT64}
 1 1 ─ %1 = (Base.slt_int)(1, b)::Bool
   └──      unless %1 goto 4
   2 ─      return 1
-  3 ─      return 1.0
+  3 ─ %4 = return 1.0
```

Comment:
NO! It seems like the bug here is that the control flow uses statement numbers still.

Reply:
Never mind, I saw you changed the printing for CodeInfo.